src/providers/routes.provider.tsx

```diff
@@ -64,73 +64,74 @@ interface RoutesContextValue {
     loadingRoutes: boolean;
     refreshRoutes: () => void;
 }
+
 // this path object provides a common place to define route pathnames
 const paths: PathObject = {
-    admin: "/admin",
-    adminViewTicket: "/admin/ticket/:ticketId",
-    adminManageEvents: "/admin/manage",
+    // admin: "/admin",
+    // adminViewTicket: "/admin/ticket/:ticketId",
+    // adminManageEvents: "/admin/manage",
     notFound: "*",
-    login: "/login",
+    // login: "/login",
     portal: "/",
-    verifyEmail: "/verify-email",
-    schedule: "/schedule",
-    networking: "/networking",
-    myTicket: "/my-ticket",
-    application: "/application",
-    submitted: "/submitted",
-    verifyRSVP: "/verify-rsvp",
-    myTeam: "/my-team",
-    joinTeam: "/join-team",
-    myApp: "/my-application",
-    ticket: "/ticket/:ticketId",
-    perks: "/perks",
+    // verifyEmail: "/verify-email",
+    // schedule: "/schedule",
+    // networking: "/networking",
+    // myTicket: "/my-ticket",
+    // application: "/application",
+    // submitted: "/submitted",
+    // verifyRSVP: "/verify-rsvp",
+    // myTeam: "/my-team",
+    // joinTeam: "/join-team",
+    // myApp: "/my-application",
+    // ticket: "/ticket/:ticketId",
+    // perks: "/perks",
 };
```

Comment:
I think we should not comment this out because we still need the basic routes and some of the components/routes redirects are based on this which causes TypeScript issues when compiling.